packages/block-editor/src/components/url-input/index.js

```diff
@@ -471,70 +393,59 @@ class URLInput extends Component {
 				role: 'option',
 				tabIndex: '-1',
 				id: `${ suggestionOptionIdPrefix }-${ index }`,
-				ref: this.bindSuggestionNode( index ),
+				ref: bindSuggestionNode( index ),
 				'aria-selected': index === selectedSuggestion,
 			};
 		};
 
-		if (
-			isFunction( renderSuggestions ) &&
-			showSuggestions &&
-			!! suggestions.length
-		) {
-			return renderSuggestions( {
+		if ( isFunction( experimentalRenderSuggestions ) ) {
```

Comment:
I know it was there from before, but is it worth it to use `lodash` instead a type check? `typeof .... === 'function'`

Reply:
Yep, I reduced the usage of lodash over all in f6a9403